sonarqube/datadog_checks/sonarqube/check.py

```diff
@@ -198,4 +198,5 @@ def metric_matcher(metric):
 
     @staticmethod
     def is_valid_metric(metric):
-        return not metric['hidden'] and metric['type'] in NUMERIC_TYPES
+        metric_re = '^(% s)' % '|'.join(BLOCKED_METRICS)
+        return not metric['hidden'] and metric['type'] in NUMERIC_TYPES and not re.match(metric_re, metric['key'])
```

Comment:
For better performance, until there are more patterns:

```suggestion
        return (
            not metric['hidden']
            and metric['type'] in NUMERIC_TYPES
            # https://github.com/DataDog/integrations-core/pull/8552
            and not metric['key'].startswith('new_')
        )
```